spec/spec_helper.cr

```diff
@@ -10,6 +11,7 @@ Db::Migrate.new(quiet: true).call
 
 Spec.before_each do
   TestDatabase.truncate
+  FileUtils.rm_r("tmp") rescue nil
```

Comment:
I don't think we should `rescue nil` because it might `rescue` a legit error. Could you rescue a specific error instead? Or maybe `rm_rf` so it deletes if it isn't there.

Also I'd suggest doing `Dir.mkdir_p("tmp")` so it is there for any test that needs it and doesn't need to manually be added

Reply:
Yeah, turns out we already had a way to do this 😂